
Showing logs for build [32mablazleon-jxpy-pr-28-pr-build-kvlp8-1[0m stage [32mfrom-build-pack[0m and container [32mstep-credential-initializer-j5krv[0m
{"level":"info","ts":1593200305.747162,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1593200305.7484262,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mablazleon-jxpy-pr-28-pr-build-kvlp8-1[0m stage [32mfrom-build-pack[0m and container [32mstep-working-dir-initializer-xwq74[0m
{"level":"info","ts":1593200306.4979806,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1593200306.4992752,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mablazleon-jxpy-pr-28-pr-build-kvlp8-1[0m stage [32mfrom-build-pack[0m and container [32mstep-place-tools[0m

Showing logs for build [32mablazleon-jxpy-pr-28-pr-build-kvlp8-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-source-ablazleon-jxpy-pr-28-pr-build-kvlp8-7pnm7[0m
{"level":"info","ts":1593200315.595477,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1593200317.1487334,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/*********/**py.git @ master in path /workspace/source"}

Showing logs for build [32mablazleon-jxpy-pr-28-pr-build-kvlp8-1[0m stage [32mfrom-build-pack[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mablazleon-jxpy-pr-28-pr-build-kvlp8-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:c7bd1dcd27a9e72ed15367642423c203d4e8b61d,28:598439c814b6284dcf3825da21dfef2d7f103efb
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git fetch origin --unshallow 598439c814b6284dcf3825da21dfef2d7f103efb: c7bd1dcd27a9e72ed15367642423c203d4e8b61d:
DEBUG: ran git fetch --unshallow origin 598439c814b6284dcf3825da21dfef2d7f103efb: c7bd1dcd27a9e72ed15367642423c203d4e8b61d: in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git branch
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git checkout master
DEBUG: ran git checkout master in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git reset --hard c7bd1dcd27a9e72ed15367642423c203d4e8b61d
DEBUG: ran git reset --hard c7bd1dcd27a9e72ed15367642423c203d4e8b61d in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git clean -fd .
DEBUG: ran clean --force -d . in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git merge 598439c814b6284dcf3825da21dfef2d7f103efb
DEBUG: ran git merge 598439c814b6284dcf3825da21dfef2d7f103efb in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git log --format=%H%x1f%an%x1f%ae%x1f%cn%x1f%ce%x1f%s%n%b%x1e c7bd1dcd27a9e72ed15367642423c203d4e8b61d..HEAD
Merged SHA 598439c814b6284dcf3825da21dfef2d7f103efb with commit message 'Update app.py' into base branch master

Showing logs for build [32mablazleon-jxpy-pr-28-pr-build-kvlp8-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-python-unittest[0m

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK

Showing logs for build [32mablazleon-jxpy-pr-28-pr-build-kvlp8-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-container-build[0m
E0626 19:38:45.415549       9 aws_credentials.go:77] while getting AWS credentials NoCredentialProviders: no valid providers in chain. Deprecated.
	For verbose messaging see aws.Config.CredentialsChainVerboseErrors
[36mINFO[0m[0002] Using dockerignore file: /workspace/source/.dockerignore 
[36mINFO[0m[0002] Retrieving image manifest python:onbuild     
E0626 19:38:46.300862       9 metadata.go:154] while reading 'google-dockercfg' metadata: http status code: 404 while fetching url http://metadata.google.internal./computeMetadata/v1/instance/attributes/google-dockercfg
E0626 19:38:46.304775       9 metadata.go:166] while reading 'google-dockercfg-url' metadata: http status code: 404 while fetching url http://metadata.google.internal./computeMetadata/v1/instance/attributes/google-dockercfg-url
[36mINFO[0m[0003] Retrieving image manifest python:onbuild     
[36mINFO[0m[0004] Built cross stage deps: map[]                
[36mINFO[0m[0004] Retrieving image manifest python:onbuild     
[36mINFO[0m[0005] Retrieving image manifest python:onbuild     
[36mINFO[0m[0007] Executing 3 build triggers                   
[36mINFO[0m[0007] Checking for cached layer gcr.io/envjenkinsk8udacitycapstone/cache:dc2a839bd6d525fd9783a6852939774c55dc14024a49397cd18ef840aae363e1... 
[36mINFO[0m[0007] Using caching version of cmd: COPY requirements.txt /usr/src/app/ 
[36mINFO[0m[0007] Checking for cached layer gcr.io/envjenkinsk8udacitycapstone/cache:8bf2abf0a84228eb1cc03999f3ee3ce7361230c16e28eae7af74285f6291b7d4... 
[36mINFO[0m[0008] Using caching version of cmd: RUN pip install --no-cache-dir -r requirements.txt 
[36mINFO[0m[0008] Checking for cached layer gcr.io/envjenkinsk8udacitycapstone/cache:29ee708c0ce94da03f9e30bf317e040d1ecd7d11b739bfbf32be3fdef498b314... 
[36mINFO[0m[0009] No cached layer found for cmd COPY . /usr/src/app 
[36mINFO[0m[0009] cmd: EXPOSE                                  
[36mINFO[0m[0009] Adding exposed port: 8080/tcp                
[36mINFO[0m[0009] Unpacking rootfs as cmd COPY . /usr/src/app requires it. 
[36mINFO[0m[0038] COPY requirements.txt /usr/src/app/          
[36mINFO[0m[0038] Found cached layer, extracting to filesystem 
[36mINFO[0m[0039] RUN pip install --no-cache-dir -r requirements.txt 
[36mINFO[0m[0039] Found cached layer, extracting to filesystem 
[36mINFO[0m[0040] COPY . /usr/src/app                          
[36mINFO[0m[0040] Resolving 121 paths                          
[36mINFO[0m[0040] Taking snapshot of files...                  
[36mINFO[0m[0040] Pushing layer gcr.io/envjenkinsk8udacitycapstone/cache:29ee708c0ce94da03f9e30bf317e040d1ecd7d11b739bfbf32be3fdef498b314 to cache now 
[36mINFO[0m[0040] COPY requirements.txt .                      
[36mINFO[0m[0040] Resolving 1 paths                            
[36mINFO[0m[0040] Taking snapshot of files...                  
[36mINFO[0m[0040] Pushing layer gcr.io/envjenkinsk8udacitycapstone/cache:104870abd6d74c9257ea3daa5b8a7cd37856df9a2180ad7355e6813edf4de844 to cache now 
[36mINFO[0m[0040] ENV PORT 8080                                
[36mINFO[0m[0040] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0040] EXPOSE 8080                                  
[36mINFO[0m[0040] cmd: EXPOSE                                  
[36mINFO[0m[0040] Adding exposed port: 8080/tcp                
[36mINFO[0m[0040] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0040] ENTRYPOINT ["python"]                        
[36mINFO[0m[0040] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0040] CMD ["app.py"]                               
[36mINFO[0m[0040] No files changed in this command, skipping snapshotting. 

Showing logs for build [32mablazleon-jxpy-pr-28-pr-build-kvlp8-1[0m stage [32mfrom-build-pack[0m and container [32mstep-postbuild-post-build[0m
no CVE provider running in the current ** namespace so skip adding image to be analysed

Showing logs for build [32mablazleon-jxpy-pr-28-pr-build-kvlp8-1[0m stage [32mfrom-build-pack[0m and container [32mstep-promote-make-preview[0m
sed -i -e "s/version:.*/version: 0.0.0-SNAPSHOT-PR-28-1/" Chart.yaml
sed -i -e "s/version:.*/version: 0.0.0-SNAPSHOT-PR-28-1/" ../*/Chart.yaml
sed -i -e "s|repository:.*|repository: gcr.io\/envjenkinsk8udacitycapstone\/**py|" values.yaml
sed -i -e "s/tag:.*/tag: 0.0.0-SNAPSHOT-PR-28-1/" values.yaml
echo "  version: 0.0.0-SNAPSHOT-PR-28-1" >> requirements.yaml
** step helm build
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Adding missing Helm repo: storage.googleapis.com https://storage.googleapis.com/chartmuseum.jenkins-x.io
Successfully added Helm repository storage.googleapis.com.
Adding missing Helm repo: jenkins-x-chartmuseum http://jenkins-x-chartmuseum:8080
Successfully added Helm repository jenkins-x-chartmuseum.
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Adding missing Helm repo: chartmuseum.jenkins-x.io http://chartmuseum.jenkins-x.io
Successfully added Helm repository chartmuseum.jenkins-x.io.

Showing logs for build [32mablazleon-jxpy-pr-28-pr-build-kvlp8-1[0m stage [32mfrom-build-pack[0m and container [32mstep-promote-jx-preview[0m
Creating a preview
Found commit author match for: ********* with email address: adrian.blazquez.leon@alumnos.upm.es
Created environment *********-**py-pr-28
Namespace **-*********-**py-pr-28 created 
expose:
  Annotations:
    helm.sh/hook: post-install,post-upgrade
    helm.sh/hook-delete-policy: hook-succeeded
  config:
    domain: 34.91.102.68.nip.io
    exposer: Ingress
    http: "true"
preview:
  image:
    repository: gcr.io/envjenkinsk8udacitycapstone/**py
    tag: 0.0.0-SNAPSHOT-PR-28-1

Updating PipelineActivities *********-**py-pr-28-1 which has status Running
Preview application is now available at: http://**py.**-*********-**py-pr-28.34.91.102.68.nip.io
